### PR TITLE
Make last OpsManager template generic. Clean up and add README.

### PR DESCRIPTION
**Pull request by niklasvincent**

### CODE REVIEW

**Comment by philmcmahon** [comment]
:+1:

File: cloudformation/mongo-opsmanager-backup.template [hunk 1]
```diff
@@ -16,6 +16,25 @@
...
+    },
+    "SshKeyFileS3Url": {
+      "Description": "S3 URL for a list of public SSH keys to use for the default ubuntu user",
+      "Type": "String",
+      "Default": ""
+    },
+    "EBSOptions": {
+      "Description": "Extra parameters to add-encrypted script",
+      "Type": "String",
+      "Default": "-t io1 -i 1000"
```
Comment (sihil) [inline cloudformation/mongo-opsmanager-backup.template:31]: Noticed you've changed the default here form 200 to 1000. Probably fine (200 is tiny for this), but thought I'd highlight it.

Comment (niklasvincent) [inline cloudformation/mongo-opsmanager-backup.template:31]: Yeah, we had some issues with the provisioned IOPS originally and 1000 was the lowest value we tried that didn't lead to a painfully slow initial sync.

We're using magnetic disk without provisioned IOPS for CODE now, but I'd love to hear your thoughts/recommendations around how much to provision for SSD in PROD.

Comment (sihil) [inline cloudformation/mongo-opsmanager-backup.template:31]: Sounds fair. I'd move to the general purpose volumes (`gp2`). That's what we switched to in CODE and they seem pretty fast and I'd say are a good compromise (they can easily exceed the IOPS you get with PIOPS, but only in a burst).

**Pull request merged**